src/routes/notification.js

```diff
@@ -0,0 +1,26 @@
+import {
+  handleMarkAsRead, handleMarkAllAsRead, handleOptInEmail, clear, handleOptOutEmail, getAll
+} from '../controllers/notificationController';
+import { checkToken } from '../middlewares/userMiddlewares';
+
+const notificationRoutes = (router) => {
+  router.route('/notification')
```

Comment:
why are this routes not documented?

Reply:
I initially didn't think there'd be a way to test them since notifications are not created directly by users but by the actions they carry out.. just realized that wasn't the case.. will add documentations where needed.

Reply:
Okay